aws/resource_aws_sns_topic.go

```diff
@@ -6,32 +6,31 @@ import (
 	"strings"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/sns"
-	"github.com/hashicorp/errwrap"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/hashicorp/terraform/helper/structure"
 )
 
 // Mutable attributes
 var SNSAttributeMap = map[string]string{
-	"arn":                                      "TopicArn",
-	"display_name":                             "DisplayName",
-	"policy":                                   "Policy",
-	"delivery_policy":                          "DeliveryPolicy",
+	"application_failure_feedback_role_arn":    "ApplicationFailureFeedbackRoleArn",
 	"application_success_feedback_role_arn":    "ApplicationSuccessFeedbackRoleArn",
 	"application_success_feedback_sample_rate": "ApplicationSuccessFeedbackSampleRate",
-	"application_failure_feedback_role_arn":    "ApplicationFailureFeedbackRoleArn",
-	"http_success_feedback_role_arn":           "HTTPSuccessFeedbackRoleArn",
-	"http_success_feedback_sample_rate":        "HTTPSuccessFeedbackSampleRate",
-	"http_failure_feedback_role_arn":           "HTTPFailureFeedbackRoleArn",
-	"lambda_success_feedback_role_arn":         "LambdaSuccessFeedbackRoleArn",
-	"lambda_success_feedback_sample_rate":      "LambdaSuccessFeedbackSampleRate",
-	"lambda_failure_feedback_role_arn":         "LambdaFailureFeedbackRoleArn",
-	"sqs_success_feedback_role_arn":            "SQSSuccessFeedbackRoleArn",
-	"sqs_success_feedback_sample_rate":         "SQSSuccessFeedbackSampleRate",
-	"sqs_failure_feedback_role_arn":            "SQSFailureFeedbackRoleArn"}
+	"arn":                                 "TopicArn",
+	"delivery_policy":                     "DeliveryPolicy",
+	"display_name":                        "DisplayName",
+	"http_failure_feedback_role_arn":      "HTTPFailureFeedbackRoleArn",
+	"http_success_feedback_role_arn":      "HTTPSuccessFeedbackRoleArn",
+	"http_success_feedback_sample_rate":   "HTTPSuccessFeedbackSampleRate",
+	"lambda_failure_feedback_role_arn":    "LambdaFailureFeedbackRoleArn",
+	"lambda_success_feedback_role_arn":    "LambdaSuccessFeedbackRoleArn",
+	"lambda_success_feedback_sample_rate": "LambdaSuccessFeedbackSampleRate",
+	"policy":                           "Policy",
+	"sqs_failure_feedback_role_arn":    "SQSFailureFeedbackRoleArn",
+	"sqs_success_feedback_role_arn":    "SQSSuccessFeedbackRoleArn",
+	"sqs_success_feedback_sample_rate": "SQSSuccessFeedbackSampleRate",
+}
```

Comment:
I kind of wish this map wasn't here at all, but there's only so much refactoring that can be done in a single PR... 😄